### ISSUE TITLE
Scheduling takes more than 3 minutes on a new cluster

### ISSUE THREAD

**Issue by vishh** [GoogleCloudPlatform/kubernetes#5956]
With a cluster built from head on `03/25/15`, scheduling takes more than three minutes. Are there any recent uncaught regressions?

```shell
monitoring-heapster-controller-lwqz1      heapster      kubernetes/heapster:v0.9        <unassigned>             name=heapster,uses=monitoring-influxdb          Pending             3 minutes
monitoring-influx-grafana-controller-6ul6t         influxdb      kubernetes/heapster_influxdb:v0.3      <unassigned>           name=influxGrafana               Pending             3 minutes
                                                  grafana      vish/heapster_grafana:canary                                                                                                                                                                     
```

**Comment by piosz** [comment]
cc @wojtek-t @fgrzadkowski

**Comment by fgrzadkowski** [comment]
@vishh How do you measure scheduling? Just assigning host or until it is reported as Running?

**Comment by vishh** [comment]
Until is assigned to a node. As the snapshot shows, the pod was unassigned
for more than 3 minutes. Restarting the scheduler seems to fix the issue.

On Thu, Mar 26, 2015 at 2:55 AM, Filip Grzadkowski <notifications@github.com

**Comment by lavalamp** [comment]
If restarting the scheduler fixed it, then you must have stepped on #6023.

**Comment by lavalamp** [comment]
This bug is confusing-- this isn't a perf problem, this is a correctness problem. Closing this as a dup of #6023.
